README.md

```diff
@@ -556,6 +556,23 @@ applications on your system. Configure your terminal to use this font:
 - **Guake**: Right Click on an open terminal and open *Preferences*. Under *Appearance* 
   tab, uncheck *Use the system fixed width font* (if not already) and select `MesloLGS NF Regular`. 
   Exit the Preferences dialog by clicking *Close*.  
+- **Alacritty**: Edit or create `~/.config/alacritty/alacritty.yml`. Append the `font` section below:
+  ```yaml
+  font:
+    normal:
+      family: "MesloLGS NF"
+      style: Regular
```

Comment:
Also this is default face to normal style, no point of specifying it here.

Reply:
It's based on the wiki entries below, which uncomment different values. I did not find a functional difference in testing.  Are you saying "font: normal: family: "MesloLGS NF"" is all that's required and the rest is implied?

[Alacritty wiki](https://github.com/alacritty/alacritty/wiki/Changing-the-default-font)
[Arch wiki](https://wiki.archlinux.org/index.php/Alacritty#Font)
[Gentoo wiki](https://wiki.gentoo.org/wiki/Alacritty#Font_configuration)

Reply:
> Are you saying "font: normal: family: "MesloLGS NF"" is all that's required and the rest is implied?

Yes, that's all that needed. Wiki's purpose is to be verbose to explain various use cases. But here specifying just font family is minimal configuration change to get expected result.

Reply:
Great, thanks. 001da9b includes this change.